src/pch.h

```diff
@@ -22,6 +22,7 @@
 
 #include "e_akv_err.h"
 #include "base64.h"
```

Comment:
This needs to be deleted - we don't need the old encoder

Reply:
May need to check up on this- for some reason the way the engine is built, all headers have to be here or else it fails to compile.